### PR TITLE
Update Gathering information for troubleshooting sites.md

### PR DESCRIPTION
**Pull request by lindsey-em**
Fixing the page formatting that was broken in migration

I've included an empty header `###` inside each `<details>` section because it was was the only way I was able to get a space between the `<summary>` and the content.

### CODE REVIEW

File: content/support/Troubleshooting/General Troubleshooting/Gathering information for troubleshooting sites.md [hunk 1]
```diff
@@ -32,107 +32,160 @@ Support.
...
+## Desktop Browsers
+<details>
+<summary>In Chrome</summary>
+  
+  ###
+  1\. In a browser page, right-click anywhere and select **Inspect Element**.
+  
+  2\. The developer tools either appear at the bottom or left side of the browser. Click the **Network** tab.
+  
+  ![HAR network tab screenshot from Chrome developer tools](/assets/images/support/gathering_har_file_network.png)
```
Comment (kodster28) [inline content/support/Troubleshooting/General Troubleshooting/Gathering information for troubleshooting sites.md:44]: Should just be `/images/support/` and then they'll resolve correctly.

Also, fwiw, direct headers are usually better than dropdowns for search crawlers (google / algolia, etc).

**Review by kodster28 — CHANGES_REQUESTED** [review]
Should just be `/images/support/` and then they'll resolve correctly.

Also, not sure I love shoving empty header tags on our pages. Maybe go with `<br/>` instead? All headers gets picked up by search crawlers (google / algolia) and might lead to not so ideal results.

Finally, you should remove the `/` from the numbers so the lists render correctly.

**Comment by pedrosousa** [comment]
> I've included an empty header `###` inside each `<details>` section because it was was the only way I was able to get a space between the `<summary>` and the content.

No need to use a hack — actually, you're missing an internal `div`. The full syntax is the following (without the `+` signs):

```diff
  <details>
  <summary>Summary here</summary>
+ <div>

  ( content here )

+ </div>
  </details>
```

If you have questions, please ask someone or check how it's being handled elsewhere.

**Comment by lindsey-em** [comment]
Quick question on this: 

`Should just be /images/support/ and then they'll resolve correctly.`

I added the /assets/ part of the uri because for me the images weren't loading in the preview. Is that expected behavior then?

**Comment by kodster28** [comment]
Oh..... that is weird. Don't know if I've seen that happen before.

**Comment by zeinjaber** [comment]
@lindsey-em, have you addressed @pedrosousa's comment?

**Comment by lindsey-em** [comment]
I am prioritizing it right now!

**Comment by lindsey-em** [comment]
Spell check fail is a false inclusive language failure, but I don't know about the GitGuardian one. I just removed backslashes after numbered lists on those files

**Comment by pedrosousa** [comment]
No need to worry about Git Guardian. It's an old false positive that got triggered when you merged production to this PR's branch.

**Comment by kodster28** [comment]
Closing this b/c the git history is pulling in a lot more changes. Think it got messed up when you merged in production.

**Comment by lindsey-em** [comment]
It was intentional, but I can make a new PR with a newly defined scope - I fixed all instances of numbering like

```
1\.
2\.
3\.

```
in the docs, after seeing the correction made here, see:

```
@lindsey-em
[Fix for](https://github.com/cloudflare/cloudflare-docs/pull/9981/commits/e7b3b697f3c5a733ba5dc0e9bab08ba7fde8b407) https://github.com/cloudflare/cloudflare-docs/pull/9981 [bundled with additional syntax fixes](https://github.com/cloudflare/cloudflare-docs/pull/9981/commits/e7b3b697f3c5a733ba5dc0e9bab08ba7fde8b407) 
[e7b3b69](https://github.com/cloudflare/cloudflare-docs/pull/9981/commits/e7b3b697f3c5a733ba5dc0e9bab08ba7fde8b407)
I used a regex query to find any other examples of n\. formatting to fix list additional list formatting issues.
```

**Pull request closed without merging**